src/foam/box/HTTPBox.js

```diff
@@ -174,6 +175,11 @@ protected class Outputter extends foam.lib.json.Outputter {
     super.outputFObject(o);
   }
 }
+  protected foam.lib.formatter.JSONFObjectFormatter getFormatter(X x) {
```

Comment:
re: ThreadLocal - yes, will add.  I'll add support in the lib.formatter package as we need this everywhere we use one.